etc/spack/defaults/packages.yaml

```diff
@@ -21,6 +21,7 @@ packages:
       blas: [openblas]
       daal: [intel-daal]
       elf: [elfutils]
+      gl: [opengl, mesa]
```

Comment:
@scottwittenburg could you order `mesa` first? In that case `vtk`/`qt` will succeed without any intervention in `packages.yaml`. I think putting `opengl` first would make more sense when `spack bootstrap` updates `packages.yaml` automatically.